sdk/eventhubs/azure-eventhubs/src/test/java/com/azure/messaging/eventhubs/SetPrefetchCountTest.java

```diff
@@ -24,6 +25,7 @@
 /**
  * Verifies we can use various prefetch options with {@link EventHubConsumer}.
  */
+@Ignore("This test creates too many resources. We need to fix this test.")
```

Comment:
I always worry about ignored tests never getting 'unignored'. Could you make sure to have an issue filed to unignore all ignored issues in event hubs?

Reply:
I had time to fix the test. It uncovered another bug in the code. Thanks for the feedback!